impeller/entity/entity_pass.cc

```diff
@@ -129,11 +129,13 @@ std::optional<Rect> EntityPass::GetElementsCoverage(
                                           Entity::RenderingMode::kSubpass);
         if (backdrop_filter) {
           auto backdrop_coverage = backdrop_filter->GetCoverage({});
-          backdrop_coverage->origin += result->origin;
           if (backdrop_coverage.has_value()) {
             if (unfiltered_coverage.has_value()) {
-              unfiltered_coverage = coverage->Union(*backdrop_coverage);
+              unfiltered_coverage = coverage.has_value()
+                                        ? coverage->Union(*backdrop_coverage)
```

Comment:
I have new functions in https://github.com/flutter/engine/pull/47183 that perform nullopt aware Union and Intersection that might simplify this code.